domains/authorization/Authorization.API/API/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -80,6 +80,17 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .IsRequired()
                         .HasColumnType("text");
 
+                    b.Property<DateTimeOffset?>("ServiceProviderTermsAcceptanceDate")
+                        .HasColumnType("timestamp with time zone");
+
+                    b.Property<bool>("ServiceProviderTermsAccepted")

```

Comment:
This seems a bit "chatgiperish" <-- self made up word. ChatGPT sometimes just spits out anti patterns.

You first tell EF: b.Property<DateTimeOffset?> and also: .HasColumnType("timestamp with time zone") which is abit double down on the same thing.

Reply:
It's copy paste from previous terms implementation

Reply:
also, once again. you are reviewing an autogenerated file.

Reply:
ahhh :D me bad.